src/main/java/com/spotify/futures/CompletableFutures.java

```diff
@@ -83,6 +113,27 @@ private CompletableFutures() {
         });
   }
 
+  private static <T> void setupFailFast(
+      FailFast failFast, CompletableFuture<? extends T>[] all, CompletableFuture<Void> allOf) {
+    ExecutionMetadata executionMeta = new ExecutionMetadata(all);
+    for (int i = 0; i < all.length; i++) {
+      all[i].whenComplete(
```

Comment:
Thic could use exceptionally since result is not used

Reply:
Wanted to use a completable future method that inspect the result rather that potentially change it (even when not getting used). But you are right, `exceptionally` would simplify the execution :+1: 

- [ ] Use `exceptionally` instead of `whenComplete`